ci/badge/gen_badge.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+##
+# Copyright (C) 2019 Samsung Electronics
+# License: LGPL-2.1
+#
+# @file gen_github_badge_svg.py
+# @brief A tool for generating a github badge image in svg format
+# @author Wook Song <wook16.song@samsung.com>
+
+import sys
+import os
+from bs4 import BeautifulSoup
```

Comment:
Please, comment the required python package in order to import bs4, requests, and svgwrite.

Reply:
@leemgs PyPi pages are added.

Reply:
Checked.